data/locale/fr-FR.ini

```diff
@@ -4,22 +4,22 @@ NDIPlugin.SourceProps.SourceName="Source"
 NDIPlugin.SourceProps.Bandwidth="Bandwidth"
 NDIPlugin.SourceProps.Sync="Synchronisation"
 NDIPlugin.SourceProps.HWAccel="Autoriser l'accélération matérielle"
-NDIPlugin.SourceProps.AlphaBlendingFix="Correction du compositing alpha (ajoute un filtre à la source)"
+NDIPlugin.SourceProps.AlphaBlendingFix="Correction de simulation de transparence (ajoute un filtre à la source)"
 NDIPlugin.SourceProps.ColorRange="Gamme de couleurs YUV"
 NDIPlugin.SourceProps.ColorRange.Partial="Partielle"
 NDIPlugin.SourceProps.ColorRange.Full="Complète"
-NDIPlugin.SourceProps.ColorSpace="Espace de couleurs YUV"
+NDIPlugin.SourceProps.ColorSpace="Espace colorimétrique YUV"
 NDIPlugin.SourceProps.Latency="Latence"
-NDIPlugin.SourceProps.Latency.Normal="Normale (sûr)"
+NDIPlugin.SourceProps.Latency.Normal="Normale (fiable)"
 NDIPlugin.SourceProps.Latency.Low="Réduite (expérimental)"
 NDIPlugin.BWMode.Highest="Haut débit"
 NDIPlugin.BWMode.Lowest="Bas débit"
 NDIPlugin.BWMode.AudioOnly="Audio seulement"
 NDIPlugin.SyncMode.NDITimestamp="Réseau"
-NDIPlugin.SyncMode.NDISourceTimecode="Timing de la source"
+NDIPlugin.SyncMode.NDISourceTimecode="Code temporel de la source"
```

Comment:
Google Translate says English to French of `Source Timing` says `Synchronisation des sources`.

`Code temporel de la source` translates back as:
> Source timecode

`Timing de la source` translates back as:
> Source Timing

Is `Code temporel de la source` really best?